ibis/expr/operations/relations.py

```diff
@@ -161,6 +161,7 @@ def values(self):
     "any_inner",
     "any_left",
     "cross",
+    "temporal",
```

Comment:
We can encode left and inner variants using `temporal_inner` and `temporal_left`.